sdk/mixedreality/mixedreality-authentication/.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
@deyaaeldeen removed them because they weren't needed, but we seem to be having trouble debugging without them lately. I don't think they hurt anything, but I'll leave what the correct guidance is to Deya. :)

Reply:
Personally, I found it handy and helpful. I know you can run any script, but I liked the VS Code integration that came with having the `launch.json` file. Just my 2 cents.

I'm happy to remove it (especially now that most of the work is done) if you'd prefer.

Reply:
@xirzec I am curios what issues are you facing with it? I can look into it.

@craigktreasure We rely on [automatic debug configuration](https://code.visualstudio.com/updates/v1_45#_automatic-debug-configurations) for debugging our tests, it is pretty easy to use and does not need any configuration. That being said, one case that feature does not support is running just the current js file (such as samples), so some of us add this simple configuration for this case: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/metricsadvisor/ai-metrics-advisor/.vscode/launch.json.

Reply:
Thanks for the suggestions @deyaaeldeen!

So, do I keep the `launch.json` or remove it?

Reply:
@craigktreasure I would remove it.

Reply:
Done!